==== triton/lib/Dialect/Triton/IR/Dialect.cpp#6 - /google/src/cloud/jreiffers/mlir_26a0b277369adc31b162b1cc38b1a712bc10c1a0_1700552908/triton/lib/Dialect/Triton/IR/Dialect.cpp ====
# action=edit type=text
--- triton/lib/Dialect/Triton/IR/Dialect.cpp	2023-10-12 01:35:16.000000000 -0700
+++ triton/lib/Dialect/Triton/IR/Dialect.cpp	2023-11-21 01:58:04.000000000 -0800
@@ -64,8 +64,7 @@
 
   /// Handle the given inlined terminator by replacing it with a new operation
   /// as necessary.
-  void handleTerminator(Operation *op,
-                        ArrayRef<Value> valuesToRepl) const final {
+  void handleTerminator(Operation *op, ValueRange valuesToRepl) const final {
     // Only return needs to be handled here.
     auto returnOp = cast<triton::ReturnOp>(op);
 
